themes/salix/layouts/partials/footer.html

```diff
@@ -0,0 +1,19 @@
+<section class="contact">
+	<div class="contact__container">
+		<h2 id="contact" class="contact__title">Contact</h2>
+		<div class="contact__content">
+			<p class="contact__message">
+				We'd love to work with you, why not drop us an email at
+			</p>
+			<a href="mailto:hello@example.net" class="contact__link"
+				>hello@example.net</a
+			>
+			<img
+				class="contact__img"
+				src="img/pigeon.png"
+				alt="a pigeon in flight possibly delivering your message."
```

Comment:
I'd just make this invisible to screen readers; the image isn't important and we probably actually don't want the content interrupted when reading by a description of it, I'd think (With a large question mark for anyone who knows better?)